docs/visualization/path/response_time.md

```diff
@@ -35,30 +35,41 @@ The following scripts generate histograms of response time. Horizontal axis of t
 # plot best-to-worst case
 plot = Plot.create_response_time_histogram_plot(path)
 plot.show()
+# or
+# plot = Plot.create_response_time_timeseries_plot(path, case='all')
+# plot.show()
 ```
 
 <prettier-ignore-start>
 !!!info
     The `output_notebook()` is needed to show figures inside Jupyter Notebook.
 <prettier-ignore-end>
 
-![response_time_default_histogram](../../imgs/response_time_default_histogram.png)
+![response_time_all_histogram](../imgs/response_time_histogram_all.png)
 
 ```python
 # plot best case
 plot = Plot.create_response_time_histogram_plot(path, case='best')
 plot.show()
 ```
 
-![response_time_best_histogram](../../imgs/response_time_best_histogram.png)
+![response_time_best_histogram](../imgs/response_time_histogram_best.png)
 
 ```python
 # plot worst case
 plot = Plot.create_response_time_histogram_plot(path, case='worst')
 plot.show()
 ```
 
-![response_time_worst_histogram](../../imgs/response_time_worst_histogram.png)
+![response_time_worst_histogram](../imgs/response_time_histogram_worst.png)
+
+```python
+# plot worst case
```

Comment:
```suggestion
# plot worst-in-input case
```

Reply:
@nabetetsu 
Fixed it in [416a00e](https://github.com/tier4/CARET_doc/pull/241/commits/416a00ebc784c5fe387b2a325711ebfeac899937).